src/main/java/com/github/mongobee/dao/ChangeEntryDao.java

```diff
@@ -112,8 +112,7 @@ public void save(ChangeEntry changeEntry) throws MongobeeConnectionException {
 
   private void verifyDbConnection() throws MongobeeConnectionException {
     if (getMongoDatabase() == null) {
-      throw new MongobeeConnectionException("Database is not connected. Mongobee has thrown an unexpected error",
-          new NullPointerException());
+      throw new MongobeeConnectionException("Database is not connected. Mongobee has thrown an unexpected error");
```

Comment:
Add a comment that the error is caused by null object. When there is no NPE passed we loose this information.